app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,481 @@ def navigation
         }
       end
-      
-      
+
+
       def content_music
         {
-          :channel_info  => {
-            :name => content_common[:name],
-            :description => content_common[:description],
-            :stats  => {
-              :items => [
+          channel_info: {
+            name: content_common[:name],
+            description: content_common[:description],
+            stats: {
+              items: [
                   {
-                    :count => "216,014",
-                    :text => t('site.channels.data-types.images'),
-                    :url => "#"
+                    count: "216,014",
+                    text: t('site.channels.data-types.images'),
+                    url: "#"
                   },
                   {
-                    :count => "102,558",
-                    :text => t('site.channels.data-types.texts'),
-                    :url => "#"
+                    count: "102,558",
+                    text: t('site.channels.data-types.texts'),
+                    url: "#"
                   },
                   {
-                    :count => "13,925",
-                    :text => t('site.channels.data-types.moving-images'),
-                    :url => "#"
+                    count: "13,925",
+                    text: t('site.channels.data-types.moving-images'),
+                    url: "#"
                   },
                   {
-                    :count => "1001",
-                    :text => t('site.channels.data-types.3d'),
-                    :url => "#"
+                    count: "1001",
+                    text: t('site.channels.data-types.3d'),
+                    url: "#"
                   },
                   {
-                    :count => "450,068",
-                    :text => t('site.channels.data-types.sound'),
-                    :url => "#"
+                    count: "450,068",
+                    text: t('site.channels.data-types.sound'),
+                    url: "#"
                   }
               ]
             },
-            :recent => {
-              :title => t('site.channels.labels.recent'),
-              :items => [
+            recent: {
+              title: t('site.channels.labels.recent'),
+              items: [
                 {
-                  :text   => "Scala museum, Milan",
-                  :url    => "#",
-                  :number => '1234' + ' ' + t('site.channels.data-types.count'),
-                  :date   => "July 2015"
+                  text: "Scala museum, Milan",
+                  url: "#",
+                  number: '1234' + ' ' + t('site.channels.data-types.count'),
+                  date: "July 2015"
                 },
                 {
-                  :text   => "Dresden Library",
-                  :url    => "#",
-                  :number => '4377' + ' ' + t('site.channels.data-types.count'),
-                  :date   => "November 2014"
+                  text: "Dresden Library",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.